monkey/monkey_island/cc/resources/root.py

```diff
@@ -29,7 +32,7 @@ def get(self, action=None):
     @jwt_required
     def get_server_info(self):
         return jsonify(
-            ip_addresses=local_ip_addresses(),
+            ip_addresses=self._local_ips,
```

Comment:
If we have a separate endpoint for this then probably we shouldn't include it here

Reply:
+1